src/box/memtx_tx.c

```diff
@@ -1859,11 +1872,7 @@ memtx_tx_handle_point_hole_write(struct space *space, struct memtx_story *story,
 			rlist_entry(item->ring.next,
 				    struct point_hole_item, ring);
 		has_more_items = next_item != item;
-
-		rlist_del(&item->ring);
-		rlist_del(&item->in_point_holes_list);
-		memtx_tx_mempool_free(item->txn, pool, item);
-
```

Comment:
Does this mean we had an index leak before this commit (no `index_unref` happened)? Can it be detected by a test?